docs/_docs/04_reference/cli-options.md

```diff
@@ -912,3 +913,10 @@ Currently only AWS S3 backend is supported.
 
 This flag can be used to drastically decrease time required for parsing Terragrunt files. The effect will only show if a lot of similar includes are expected such as the root terragrunt.hcl include.
 NOTE: This is an experimental feature, use with caution.
+
+### terragrunt-include-module-prefix
+
+**CLI Arg**: `--terragrunt-include-module-prefix`
```

Comment:
Noticed that `terragrunt-include-module-prefix` is not listed in the help output
```
$ terragrunt --help | grep terragrunt-include-module-prefix
$
```

Reply:
Included the info in help output. Is there any other place the help should be updated?